docs/components/rule.md

```diff
@@ -1,42 +1,104 @@
-# Rule
```

Comment:
There is no mention of the newly added field `partial_response_strategy`, it's values, overriding the value set fort he group with value of the single rule/alert of it and meaning what it does.

Is that intentional like `hidden-flag` ?

Reply:
no, will add